templates/components/itilobject/timeline/form_solution.html.twig

```diff
@@ -168,7 +168,7 @@
                      {{ fields.dropdownField(
                         'SolutionType',
                         'solutiontypes_id',
-                        subitem.fields['solutiontypes_id'],
+                        0,
```

Comment:
This change would make it appear as if nothing is selected when editing a solution even if a choice was made and if the solution is saved, it would set unset the type.
The template field has the value set to 0 statically because it is not a field that gets stored in the database. It is only used while the form is open to change the other fields.

The easiest solution seems to be:
```suggestion
                        subitem.fields['solutiontypes_id'] ?? 0,
```

This would set the value to 0 if the current value is null or not set.

Reply:
Great, is a good idea I didn't think about editing, thank you